I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 13, in script
    $ povname = William
  File "game/script.rpy", line 13, in <module>
    $ povname = William
NameError: name 'William' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 13, in script
    $ povname = William
  File "renpy/ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 13, in <module>
    $ povname = William
NameError: name 'William' is not defined

Windows-7-6.1.7601-SP1 AMD64
Ren'Py 7.5.3.22090809
Nekrós 1.0
Mon Mar 27 21:12:06 2023
